File: java/program20.java
import java.util.*;

class Number
{
    public void Display(int iNo)
    {
        if(iNo < 0)
        {
            iNo = -iNo;
        }
        
        for(int iCnt = iNo; iCnt >= 1; iCnt--)
        {
            System.out.print(iCnt+"\t");
        }
        System.out.println();
    }
}

class program20
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Frequency");
        int iValue = sobj.nextInt();

        Number obj = new Number();
        obj.Display(iValue);
    }
}

File: cpp/program20.cpp
// program100.c

#include<iostream>
using namespace std;

class Number
{
    public:
       int iNo;

    Number(int X)
    {
        iNo = X;
    }

    void Display()
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            cout<<iCnt<<"\t";
        }
        cout<<"\n";
    }
};

int main()
{
    int iValue = 0;

    cout<<"Enter number"<<"\n";
    cin>>iValue;

    Number nobj(iValue);
    nobj.Display();

    return 0;
}